Dockerfile.monitor

```diff
@@ -0,0 +1,18 @@
+FROM registry.access.redhat.com/ubi9/go-toolset:1.22.7 AS builder
+
+USER root
+COPY ./VERSION /workdir/VERSION
+COPY ./.git /workdir/.git
+COPY ./src/runtime /workdir/src/runtime
+WORKDIR /workdir/src/runtime
+RUN CGO_ENABLED=1 GOFLAGS=-tags=strictfipsruntime make monitor
+
+FROM registry.access.redhat.com/ubi9/ubi-minimal:latest
+COPY --from=builder /workdir/src/runtime/kata-monitor /usr/bin/kata-monitor
+
+# Add only required capabilities for the monitor
+RUN chmod u-s /usr/bin/kata-monitor
+RUN setcap "cap_dac_override+eip" /usr/bin/kata-monitor
```

Comment:
What's the purpose of this? Can you run it in the builder step?

Reply:
Yes the monitor needs some extra super powers. This is typically something that only needs to be done at install time.

Reply:
It does apparently - just tried it, and the resulting image shows the same caps as the previous build.
```bash
$ podman run --entrypoint= -ti b75bbefb115579f04a5cb8bf8771d18aa2133974b2b64c8abb41e1223f96e715 bash
[root@72c59c1a3f60 /]# ls -alF /usr/bin/kata-monitor 
-rwxr-xr-x. 1 root root 47216208 Jan 23 15:49 /usr/bin/kata-monitor*
[root@72c59c1a3f60 /]# getcap /usr/bin/kata-monitor 
/usr/bin/kata-monitor cap_dac_override=eip
```

Reply:
I've added a commit to show it. Have a look and tell me - we can drop it or squash it, either way if fine with me :-)